plugins/sqlfluff-templater-dbt/sqlfluff_templater_dbt/templater.py

```diff
@@ -515,10 +524,7 @@ def make_template(in_str):
                     "dbt templater compilation failed silently, check your "
                     "configuration by running `dbt compile` directly."
                 )
-
-            with open(fname) as source_dbt_model:
-                source_dbt_sql = source_dbt_model.read()
-
+            source_dbt_sql = in_str
```

Comment:
This is the fix -- use `in_str` because it was read using the specified encoding, as noted in the docstring above.